### ISSUE TITLE
feedback on documentation

### ISSUE THREAD

**Issue by nlooije** [OpenLabTools/Microscope#7]
I hope you won't take this as negative feedback on an otherwise awesome project but i'd like to use this issue to identify some discrepancies with the documentation.

1. The Bill of Materials in the 'useful information' and 'step-by-step' guide differ for some important materials. Also using the BoM from the step-by-step guide will e.g. not get you enough T-brackets to complete the build.
2. The instructions for the linear stage towards the end of document require more specific instructions. From the text and the pictures it is still too vague how certain parts should be attached. Also is there a template for making holes into the perspex?
3. There is no documentation on the installation of the Arduino and/or RPi software. E.g. several Adafruit libraries are required to be installed. Also no mention is made anywhere of a Touchscreen in the documentation but the Arduino software requires a Touchscreen library. How is this Touchscreen used?

I may append to this list as i find more while i am finishing my build...

**Comment by JamesRitchie** [comment]
You're right, the documentation is definitely a problem. I've been intermittently writing some better documentation for the software parts, so I'll see what I can put together, but it's been a few years since I was an undergraduate and working on the project.

If you have any other questions let me know, I'll try and answer them or get in touch with the people who might know.
